textlists.py

```diff
@@ -0,0 +1,19 @@
+# data source: project gutenberg
+# printing sherlock holmes full text and make them as one list of texts
+
+import pickle
+import requests
+
+
+adventure_of_sherlock_holmes = requests.get('http://www.gutenberg.org/ebooks/1661.txt.utf-8').text
+print(adventure_of_sherlock_holmes)
+
+
+# Save data to a file
+f = open('adventure_of_sherlock_holmes.pickle', 'wb')
+pickle.dump(adventure_of_sherlock_holmes, f)
+f.close()
+
+# Load data from a file
+input_file = open('adventure_of_sherlock_holmes.pickle', 'rb')
+reloaded_copy_of_texts = pickle.load(input_file)
```

Comment:
Nice work separating data fetching functionality into a separate file so you don't have to do this every time you run an analysis.